pyomo/contrib/pyros/master_problem_methods.py

```diff
@@ -100,6 +102,23 @@ def construct_master_feasibility_problem(model_data, config):
     """
 
     model = model_data.master_model.clone()
+
+    # obtain mapping from master problem to master feasibility
+    # problem variables
+    varmap_name = unique_component_name(
+        model_data.master_model,
+        'pyros_var_map',
+    )
+    setattr(model_data.master_model, varmap_name,
+            list(model_data.master_model.component_data_objects(Var)))
```

Comment:
Noted. Should I make this change here? If I understand correctly, `master_model` has no parent, such that all variables are in the active tree rooted here.

Reply:
Oh, oops, I may have mislead you. The fact that it has no parent doesn't necessarily mean that all the variables are in the active tree (they could be on deactivated Blocks, for example), but pyros *makes* `master_model`, right? So this is actually in your control at this point, I think. I just brought it up because when you initially collect the variables from the user-defined model, you likely want to use `get_vars_from_components`.